Tasks/MavenV3/task.json

```diff
@@ -17,7 +17,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 3,
-        "Minor": 152,
+        "Minor": 153,
```

Comment:
since we're still in sprint 152 just update the patch number instead

Reply:
oops, thanks